socialhome/streams/app/components/StreamElement.vue

```diff
@@ -34,14 +34,16 @@ export default Vue.component("stream-element", {
         contentId: {type: Number, required: true},
     },
     computed: {
-        content(){
+        content() {
```

Comment:
How did this pass through my rigorous code review??? :P

Reply:
<kbd>ctrl</kbd> + <kbd>maj</kbd> + <kbd>l</kbd> FTW :D